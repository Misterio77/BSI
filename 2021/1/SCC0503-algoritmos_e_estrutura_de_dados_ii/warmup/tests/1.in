data/reg.dat
1
