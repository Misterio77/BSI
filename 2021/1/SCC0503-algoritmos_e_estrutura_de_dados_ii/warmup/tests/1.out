nUSP: 1
Nome: Winter Y. Hodges
Curso: Engenharia da Computacao
Nota: 5.00

nUSP: 2
Nome: Doris S. Mathis
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 3
Nome: Vernon U. Dunlap
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 4
Nome: Duncan D. Peters
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 5
Nome: Felix L. Landry
Curso: Engenharia da Computacao
Nota: 1.00

nUSP: 6
Nome: Keane P. Knapp
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 7
Nome: Linus Q. Mccoy
Curso: Estatistica
Nota: 4.00

nUSP: 8
Nome: Quyn T. Rowland
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 9
Nome: Geraldine V. Reilly
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 10
Nome: Knox V. Floyd
Curso: Ciencia da Computacao
Nota: 10.00

nUSP: 11
Nome: Deacon V. Olsen
Curso: Ciencia da Computacao
Nota: 2.00

nUSP: 12
Nome: Mara P. Schneider
Curso: Engenharia da Computacao
Nota: 3.00

nUSP: 13
Nome: Patience V. Bullock
Curso: Engenharia da Computacao
Nota: 8.00

nUSP: 14
Nome: Charles E. Snider
Curso: Engenharia da Computacao
Nota: 3.00

nUSP: 15
Nome: Mariko D. Gaines
Curso: Estatistica
Nota: 6.00

nUSP: 16
Nome: Nola D. Rocha
Curso: Engenharia da Computacao
Nota: 4.00

nUSP: 17
Nome: Brennan C. Zimmerman
Curso: Ciencia da Computacao
Nota: 4.00

nUSP: 18
Nome: Aurelia S. Hess
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 19
Nome: Cassidy E. Mack
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 20
Nome: Shoshana B. Carey
Curso: Estatistica
Nota: 2.00

nUSP: 21
Nome: Myra Y. Bailey
Curso: Sistemas de Informacao
Nota: 3.00

nUSP: 22
Nome: Garth J. Shepard
Curso: Estatistica
Nota: 6.00

nUSP: 23
Nome: Maile I. Henry
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 24
Nome: Sigourney H. Garrett
Curso: Estatistica
Nota: 9.00

nUSP: 25
Nome: Aaron W. Dean
Curso: Estatistica
Nota: 3.00

nUSP: 26
Nome: Brittany P. Cline
Curso: Engenharia da Computacao
Nota: 8.00

nUSP: 27
Nome: Amy L. Witt
Curso: Sistemas de Informacao
Nota: 6.00

nUSP: 28
Nome: Brenna R. Nguyen
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 29
Nome: Aladdin J. Henson
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 30
Nome: Allen M. Henry
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 31
Nome: Jade W. Richardson
Curso: Sistemas de Informacao
Nota: 4.00

nUSP: 32
Nome: Alfonso J. Decker
Curso: Ciencia da Computacao
Nota: 9.00

nUSP: 33
Nome: Jana O. Carver
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 34
Nome: Ora G. Joyce
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 35
Nome: Sade E. Horton
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 36
Nome: Quentin I. Wilder
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 37
Nome: Kennedy K. Winters
Curso: Ciencia da Computacao
Nota: 2.00

nUSP: 38
Nome: Jack G. Juarez
Curso: Estatistica
Nota: 1.00

nUSP: 39
Nome: Kimberley R. Phelps
Curso: Ciencia da Computacao
Nota: 0.00

nUSP: 40
Nome: Jennifer O. Dalton
Curso: Estatistica
Nota: 3.00

nUSP: 41
Nome: Xaviera N. Mclean
Curso: Ciencia da Computacao
Nota: 5.00

nUSP: 42
Nome: Violet V. Potter
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 43
Nome: Buckminster F. Gallegos
Curso: Engenharia da Computacao
Nota: 6.00

nUSP: 44
Nome: Martha S. Collier
Curso: Sistemas de Informacao
Nota: 8.00

nUSP: 45
Nome: Jordan X. Hill
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 46
Nome: Hadassah T. Carver
Curso: Engenharia da Computacao
Nota: 5.00

nUSP: 47
Nome: Ginger J. Nielsen
Curso: Ciencia da Computacao
Nota: 4.00

nUSP: 48
Nome: Jin K. Pena
Curso: Engenharia da Computacao
Nota: 9.00

nUSP: 49
Nome: Maite R. Klein
Curso: Sistemas de Informacao
Nota: 2.00

nUSP: 50
Nome: Rachel X. Dorsey
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 51
Nome: Rajah C. Montgomery
Curso: Ciencia da Computacao
Nota: 10.00

nUSP: 52
Nome: Garrison N. Montoya
Curso: Estatistica
Nota: 6.00

nUSP: 53
Nome: Maxwell R. Sherman
Curso: Sistemas de Informacao
Nota: 6.00

nUSP: 54
Nome: Ginger C. Stout
Curso: Ciencia da Computacao
Nota: 0.00

nUSP: 55
Nome: Daryl W. Kelley
Curso: Engenharia da Computacao
Nota: 7.00

nUSP: 56
Nome: Kuame M. Buckner
Curso: Ciencia da Computacao
Nota: 6.00

nUSP: 57
Nome: Davis I. Carr
Curso: Estatistica
Nota: 1.00

nUSP: 58
Nome: Selma I. Boyd
Curso: Ciencia da Computacao
Nota: 0.00

nUSP: 59
Nome: Aaron B. Mathis
Curso: Engenharia da Computacao
Nota: 7.00

nUSP: 60
Nome: Lara M. Yates
Curso: Engenharia da Computacao
Nota: 0.00

nUSP: 61
Nome: Honorato W. Mckenzie
Curso: Sistemas de Informacao
Nota: 5.00

nUSP: 62
Nome: Drake K. Browning
Curso: Estatistica
Nota: 6.00

nUSP: 63
Nome: Aaron W. Atkinson
Curso: Engenharia da Computacao
Nota: 2.00

nUSP: 64
Nome: Camden Z. Walters
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 65
Nome: Dai H. Lowery
Curso: Ciencia da Computacao
Nota: 0.00

nUSP: 66
Nome: Russell H. Price
Curso: Estatistica
Nota: 8.00

nUSP: 67
Nome: Armand S. Finch
Curso: Engenharia da Computacao
Nota: 10.00

nUSP: 68
Nome: Robert S. Mckee
Curso: Estatistica
Nota: 10.00

nUSP: 69
Nome: Carter J. Kerr
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 70
Nome: Velma L. Love
Curso: Estatistica
Nota: 6.00

nUSP: 71
Nome: Claudia T. Valdez
Curso: Engenharia da Computacao
Nota: 0.00

nUSP: 72
Nome: Grady E. Dodson
Curso: Ciencia da Computacao
Nota: 2.00

nUSP: 73
Nome: Eden C. Allison
Curso: Ciencia da Computacao
Nota: 8.00

nUSP: 74
Nome: Nyssa I. Powell
Curso: Estatistica
Nota: 1.00

nUSP: 75
Nome: Henry H. Cohen
Curso: Estatistica
Nota: 10.00

nUSP: 76
Nome: Charlotte G. Craig
Curso: Estatistica
Nota: 7.00

nUSP: 77
Nome: Nyssa L. Powell
Curso: Ciencia da Computacao
Nota: 10.00

nUSP: 78
Nome: Harlan O. Allen
Curso: Engenharia da Computacao
Nota: 3.00

nUSP: 79
Nome: Jonah N. Dunn
Curso: Estatistica
Nota: 3.00

nUSP: 80
Nome: Brody Q. Carpenter
Curso: Ciencia da Computacao
Nota: 1.00

nUSP: 81
Nome: Gage R. Cardenas
Curso: Estatistica
Nota: 0.00

nUSP: 82
Nome: Lewis T. Buchanan
Curso: Sistemas de Informacao
Nota: 7.00

nUSP: 83
Nome: Joseph K. Langley
Curso: Sistemas de Informacao
Nota: 10.00

nUSP: 84
Nome: Chaim Z. Rollins
Curso: Engenharia da Computacao
Nota: 4.00

nUSP: 85
Nome: Ferdinand B. Bass
Curso: Estatistica
Nota: 7.00

nUSP: 86
Nome: Martena W. Herrera
Curso: Estatistica
Nota: 3.00

nUSP: 87
Nome: Rigel N. Morales
Curso: Estatistica
Nota: 6.00

nUSP: 88
Nome: Ulla G. Knowles
Curso: Sistemas de Informacao
Nota: 9.00

nUSP: 89
Nome: Magee H. Cantrell
Curso: Ciencia da Computacao
Nota: 8.00

nUSP: 90
Nome: Ahmed H. Brooks
Curso: Estatistica
Nota: 5.00

nUSP: 91
Nome: Moses X. Wynn
Curso: Ciencia da Computacao
Nota: 5.00

nUSP: 92
Nome: Steel I. Combs
Curso: Sistemas de Informacao
Nota: 4.00

nUSP: 93
Nome: Gail Q. Downs
Curso: Estatistica
Nota: 3.00

nUSP: 94
Nome: Barclay O. Barber
Curso: Estatistica
Nota: 7.00

nUSP: 95
Nome: Aubrey S. Campos
Curso: Engenharia da Computacao
Nota: 6.00

nUSP: 96
Nome: Fleur E. Larsen
Curso: Estatistica
Nota: 3.00

nUSP: 97
Nome: Chester C. Spencer
Curso: Ciencia da Computacao
Nota: 7.00

nUSP: 98
Nome: Erica Z. Lang
Curso: Sistemas de Informacao
Nota: 0.00

nUSP: 99
Nome: Brielle D. Buckner
Curso: Engenharia da Computacao
Nota: 5.00

nUSP: 100
Nome: Ora K. Boyd
Curso: Estatistica
Nota: 6.00
